modules/custom/drupalmel_core/drupalmel_core.install

```diff
@@ -81,6 +81,7 @@ function drupalmel_core_update_7000() {
   $feeds = array(
     'Kattekrab'   => 'http://kattekrab.net/taxonomy/term/322/feed',
     'Realityloop' => 'http://realityloop.com/tags/drupal-planet/feed',
+    'Flink' => 'http://flink.com.au/taxonomy/term/1/feed',
```

Comment:
Nope, that's all the stuff you don't need :)

Lines https://github.com/mattwithoos/drupalmel/blob/flinkfeed/modules/custom/drupalmel_core/drupalmel_core.install#L87 - https://github.com/mattwithoos/drupalmel/blob/flinkfeed/modules/custom/drupalmel_core/drupalmel_core.install#L92 only, but with 'Flink' and the URL instead of the variables.